luatest/runner.lua

```diff
@@ -154,8 +155,11 @@ function Runner.parse_cmd_line(args)
         elseif arg:sub(1,1) == '-' then
             error('Unknown option: ' .. arg)
         elseif arg:find('/') then
```

Comment:
For me, it is not clear why trailing slash is mandatory for paths. Probably we should use `fio.path.exists(arg)` to decide whether it is a path or not.

Reply:
The trailing slash is not mandatory, a path can be like that `./path`. If you provide just `foo` (without the slash), luatest treats it as a group name. That is luatest will try to run all tests from `foo` group.

Reply:
Or `fio.stat(<...>)` to distinguish a regular file from a directory. It would have more expected behaviour, I guess.

However it looks as out of #175 scope.

Reply:
Guys, I am OK with your suggestions, but it is really out of the scope of the issue that is going to be fixed by this PR. Let's continue this discussion in a separate issue.